release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2233 @@
+This bug fix enhances the `new-app` circular dependency code to account for the
+`ImageStreamImage` type. The unnecessary warning no longer appears.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1422378[*BZ#1422378*])
+
+*Command Line Interface*
+
+* Previously, pod headers were only being printed once for all sets of pods when
+listing pods from multiple nodes. Executing `oadm manage-node <node-1> <node-2> ...
+--evacuate --dry-run` with multiple nodes would print the same output multiple
+times (once per each specified node). Therefore, users would see inconsistent or
+duplicate pod information. This bug fix resolves the issue.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1390900[*BZ#1390900*])
+
+* The `--sort-by` in the `oc get` command fails when any object in the list
+contains an empty value in the field used to sort, causing a failure. With this
+bug fix, empty fields in `--sort-by` are now correctly handled. The output of
+`oc get` is printed correctly and empty fields are considered in sorting.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1409878[*BZ#1409878*])
+
+* A Golang issue (in versions up to 1.7.4) adds an overhead of around four seconds
+to the TLS handshake on macOS. Therefore, the `oc` calls time out
+intermittently on macOS. This bug fix backported the existing fix to 1.7.5 and
+upgraded the Golang that we use to build `oc` to that version. The TLS handshake
+time is now reduced by about four seconds on macOS.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1435261[*BZ#1435261*])
+
+* When the master configuration specified a default `nodeSelector` for the
+cluster, test projects created by `oadm diagnostics` NetworkCheck got this
+`nodeSelector` and, therefore, the test pods were also confined to this
+`nodeSelector`. NetworkCheck test pods could only be scheduled on a subset of
+nodes, preventing the diagnostic covering the entire cluster; in some clusters,
+this might even result in too few pods running for the diagnostic to succeed
+even if the cluster health is fine. NetworkCheck now creates the test projects
+with an empty `nodeSelector` so they can land on any schedulable node. The
+diagnostic should now be more robust and meaningful.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1459241[*BZ#1459241*])
+
+*Installer*
+
+* OpenShift Ansible facts were splitting a configuration parameter incorrectly.
+Therefore, invalid `NO_PROXY` strings were generated and inserted into user
+*_sysconfig/docker_*  files. The logic that generates the NO_PROXY strings was
+reviewed and fixed. Valid Docker `NO_PROXY` settings are enerated and inserted
+into the *_sysconfig/docker_*  file now.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1414748[*1414748*])
+
+* The OpenShift CA redeployment playbook
+(*_playbooks/byo/openshift-cluster/redeploy-openshift-ca.yml_*) would fail to
+restart services if certificates were previously expired. Service restarts are
+now skipped within the OpenShift CA redeployment
```

Comment:
I'd consider making this singular, "Ansible Playbook Bundle".
--

When de-provisioning an Ansible Playbook Bundle (APB) via the